unity/Assets/Scripts/BaseFPSAgentController.cs

```diff
@@ -2662,16 +2669,27 @@ public void InitialRandomSpawn(ServerAction action)
             }
         }
 
-        public void GetShortestPath(Vector3 position, Vector3 rotation, string objectType = null, string objectId = null) {
-            getShortestPath(objectType, objectId, position, Quaternion.Euler(rotation));
+        public void GetShortestPath(
+            Vector3 position,
+            Vector3 rotation,
+            string objectType = null,
+            string objectId = null,
+            float allowedError = 0.01f
```

Comment:
We have the default of 0.01 for allowedError in both python and Unity.  I'd recommend the default occur at the lowest layer (Unity) and just default None on the Python side.

Reply:
The downside of doing this is that it's not immediately clear to the user what the default value is, they would have to look at the Unity code directly. I could specify that we use a default value of 0.01 in the python documentation but I feel that this is strictly worse as now we could change things on the Unity side and forget to change them on the python side.

I would make it a required parameter at the Unity level but this would break backwards compatibility.

Reply:
Here are a few options - 
1) make an overloaded version that has allowedError required (use the default from python).  The method without allowedError would have the backwards compatible behavior.  
2) Default allowedError to 0.0 if that would preserve existing behavior.  
3) Finally, just leave as is if you think the new behavior is better

Reply:
👍 Made `allowedError` default to a small positive number (worried about it being exactly 0 due to possible floating point issues) in Unity and the python documentation simply says that Unity will choose some small positive number so people (theoretically) should not depend on this being consistent.